### ISSUE TITLE
Unnecessary not Python 3 conditional, in twisted/web/client.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7895]
|<img alt="edorum2's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| edorum2 reported|
|-|-|
|Trac ID|trac#7895|
|Type|enhancement|
|Created|2015-05-12 13:56:09Z|
|Branch|https://github.com/twisted/twisted/tree/t.web.client-py3-imports-7895|

In twisted/web/client.py there is the following code:
if not _PY3:
    from twisted.web._newclient import Request, Response, HTTP11ClientProtocol
    from twisted.web._newclient import ResponseDone, ResponseFailed
    from twisted.web._newclient import RequestNotSent, RequestTransmissionFailed
    from twisted.web._newclient import (
        ResponseNeverReceived, PotentialDataLoss, _WrapperException)

But since twisted.web._newclient was ported to Python 3 in newclient-py3-7485. This conditional just breaks code for people that want to use the partial port of client.py on Python 3. 

Attachments:

* [7895-1.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/3ba/3bafd762ec7eec62f6cd7ce5bfc6bc003935fc13/7a57bbbc505c07603270e70f04d392169949e71d.patch) (1104 bytes) - added by edorum2 on 2015-07-20 09:08:06Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7895 7895
type__enhancement enhancement
reporter__edorum2 edorum2
priority__normal normal
milestone__ 
branch__branches_t_web_client_py3_imports_7895 branches/t.web.client-py3-imports-7895
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1431438969740216 1431438969740216
changetime__1437555154629248 1437555154629248
version__None None
owner__edorum2 edorum2

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="edorum2's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|edorum2 removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [45282]) Branching to t.web.client-py3-imports-7895.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to edorum2|
|-|-|

Changes look good... the _PY3 no longer needs to be imported and this cause the pyflakes tests to fail.

since [#7485](https://github.com/twisted/twisted/issues/7485) was .misc I guess that is ok to also have this patch as .misc.

I have fixed he _PY3.

Will sent the patch again to buildbot and if all is green I will merge it.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

(In [45285]) Merge t.web.client-py3-imports-7895: Remove Python 3 conditional import in twisted.web.client.

Author: edorum2
Reviewer: adiroiban
Fixes: [#7895](https://github.com/twisted/twisted/issues/7895)
